src/qt/sendcoinsdialog.cpp

```diff
@@ -346,6 +346,13 @@ void SendCoinsDialog::on_sendButton_clicked()
         questionString.append("</span>");
     }
 
+    if (prepareStatus.m_fallback_fee_used) {
+        questionString.append("<hr /><span style='color:#aa0000;'>");
+        questionString.append(tr("WARNING")+":</span> <span>");
+        questionString.append(tr("A static fallback transaction fee has been used due to missing fee estimations. Please consider waiting a few blocks until fee estimation is ready. Your transaction will very likely get stuck or you overpay in fees."));
```

Comment:
s/estimations/estimation.

Reply:
I would reorder the sentenses: 1, 2, 3 -> 3, 1, 2. Also, avoid you/your?

Reply:
I think the order is correct (cause, fix, possible issue).